src/api/sanity/sanity.ts

```diff
@@ -0,0 +1,16 @@
+import imageUrlBuilder from '@sanity/image-url';
+import { SanityImageSource } from '@sanity/image-url/lib/types/types';
+import { createClient } from 'next-sanity';
+
+export const client = createClient({
+  projectId: 'fgi7eh1v',
+  dataset: 'staging',
```

Comment:
Are you still using this to access Sanity?
The issue here is that we have `staging` that is hardcoded.
The `useSanity` hook takes care of this. I don't think we need this `/api/sanity`.